### ISSUE TITLE
Settings page is very slow

### ISSUE THREAD

**Issue by codyjames** [verbb/postie#63]
**Description**
The settings page loads suuuuuper slow for me. Not sure if this is the case for everyone or not. One thing to note is that we have 30k products. I see you are listing a count of products on this page so perhaps that's causing this to slow down.

I also occasionally get ```Unknown: Input variables exceeded 1000. To increase the limit change max_input_vars in php.ini.``` which I can adjust, but may indicate a bigger issue.

**Steps to reproduce**
1. Install Postie
2. Go to the Postie settings page

**Additional info**
- Plugin version: 2.4.7
- Craft version: 3.6.16
- Multi-site: No

**Comment by engram-design** [comment]
Good point. I've just limited this query to 20 variants which should help. To get the fix early, change your `verbb/postie` requirement in `composer.json` to:

```
"require": {
  "verbb/postie": "dev-craft-3 as 2.4.7",
  "...": "..."
}
```

Then run `composer update`. Interested to know if that'll fix things.

As for the `max_input_vars` that's _somewhat_ to be expected, due to how we're storing Postie provider settings in project config - all on a single page. The next few versions will change this, to not only fix this issue, but add provider instances, so you can have more than one shipping provider setup.

**Comment by codyjames** [comment]
Thanks, @engram-design! Works much better now.

**Comment by engram-design** [comment]
Fixed in [2.4.8](https://github.com/verbb/postie/releases/tag/2.4.8)

**Comment by engram-design** [comment]
This is also improved in [4.0.0](https://github.com/verbb/postie/releases/tag/4.0.0) with provider instances.
